reactor3/src/main/java/com/linecorp/armeria/common/reactor3/RequestContextAccessor.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.reactor3;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.internal.common.RequestContextUtil;
+
+import io.micrometer.context.ThreadLocalAccessor;
+
+/**
+ * TBD.
+ */
+public final class RequestContextAccessor implements ThreadLocalAccessor<RequestContext> {
```

Comment:
Thanks for your comment in holiday 🙇‍♂️ 
IMHO, it does not work well.  Because `micro-meter:context-propagation` is integrated on `Reactor` > 3.5 only. 

`micro-meter:context-propagation` produce `ContextSnapshot` to propagate context. 
However, `ContextSnapshot` now use `Reactor` contexts only to restore `ThreadLocal``s state. 
Thus, `micro-meteR:context-propagation` does not work well without `Reactor` > 3.5

Reply:
After i left comments above, i investigate internal of `micro-meter:context-propagation`.

> It could be useful for any applications that use micrometer-context-propagation.

As you said, it seems promised. 
If application implement suitable `source context` like `Reactor Context` and integrate it to their source code, other application can use `micro-meter:context-propagation`. 
(if you want to know more detail, please refer to this [code](https://github.com/micrometer-metrics/context-propagation/blob/b42c0049e46121e4e3419762895a26098eae4321/context-propagation/src/main/java/io/micrometer/context/DefaultContextSnapshotFactory.java#L107-L132))

May i move this code from `:reactor3:` to `:core:`, right now? 
or wait until major comment are resolved?

Reply:
Let's move it to `:core`. I guess IDEs will do that job without causing much trouble :smile:

Reply:
Thanks for your comment 🙇‍♂️ 
I moved it `:core`!